scheduler.go

```diff
@@ -280,10 +279,17 @@ func (j *Job) GetAt() string {
 	return fmt.Sprintf("%d:%d", j.atTime/time.Hour, (j.atTime%time.Hour)/time.Minute)
 }
 
-// From schedules the next run of the job
-func (s *Scheduler) From(t *time.Time) *Scheduler {
-	currentJob := s.getCurrentJob()
-	currentJob.nextRun = *t
+// StartAt schedules the next run of the job
+func (s *Scheduler) StartAt(t time.Time) *Scheduler {
+	s.getCurrentJob().nextRun = t
+	return s
+}
+
+// NextTick returns a pointer to a time that will run at the next tick
```

Comment:
```suggestion
// StartImmediately sets the jobs next run to now and the startsImmediately bool to true
// This will cause the scheduler to run the job immediately when it is started
```